### PR TITLE
Add GameBoardTest class and a method to prove moveLeft method

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

**Comment by jpgonzales27** [comment]
I think it's the same problem that happened to Raymundo last night

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 1]
```diff
@@ -56,10 +56,10 @@ public void init() {
                 add(label);
             }
         }
-        spaceShip();
+        /* spaceShip();
         spaceAlienGroup();
         addKeyListener(this);
-        start();
+        start(); */
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:62]: maybe this commented block of code should be removed

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 2]
```diff
@@ -232,4 +229,24 @@ public void keyPressed(final KeyEvent e) {
             switchBullet = true;
         }
     }
+
+    /** */
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:233]: The description of the methods is missing

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 3]
```diff
@@ -232,4 +229,24 @@ public void keyPressed(final KeyEvent e) {
...
+    public void executeMoveLeft(final int keyCode) {
+        if (keyCode == KeyEvent.VK_LEFT) {
+            cleanSpaceShip();
+            spaceship.moveLeft();
+            refreshSpaceShip();
+        }
+    }
+
+    /** */
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:242]: The description of the methods is missing

File: src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java [hunk 4]
```diff
@@ -232,4 +229,24 @@ public void keyPressed(final KeyEvent e) {
...
+    public Spaceship getSpaceship() {
+        return spaceship;
+    }
+
+    /** */
```
Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/GameBoard.java:247]: The description of the methods is missing

**Review by jpgonzales27 — APPROVED** [review]

**Review by edsonarios — APPROVED** [review]

**Review by danielaSantaCruz — APPROVED** [review]

**Pull request merged**